tests/emscripten_runner.js

```diff
@@ -1,109 +1,115 @@
-const { opendir } = require('node:fs/promises');
-const { loadPyodide } = require('pyodide');
+const {opendir} = require('node:fs/promises')
+const {loadPyodide} = require('pyodide')
+const path = require('path')
 
-async function findWheel(distDir) {
-  const dir = await opendir(distDir);
+async function find_wheel(dist_dir) {
+  const dir = await opendir(dist_dir)
   for await (const dirent of dir) {
-    if (dirent.name.endsWith('whl')) {
-      return dirent.name;
+    if (dirent.name.endsWith('.whl')) {
+      return path.join(dist_dir, dirent.name)
     }
   }
 }
 
-const pkgDir = process.argv[2];
-const distDir = pkgDir + '/dist';
-const testDir = pkgDir + '/tests';
-
-function make_tty_ops(stream){
+function make_tty_ops(stream) {
   return {
     // get_char has 3 particular return values:
     // a.) the next character represented as an integer
     // b.) undefined to signal that no data is currently available
     // c.) null to signal an EOF
     get_char(tty) {
       if (!tty.input.length) {
-        var result = null;
-        var BUFSIZE = 256;
-        var buf = Buffer.alloc(BUFSIZE);
-        var bytesRead = fs.readSync(process.stdin.fd, buf, 0, BUFSIZE, -1);
+        var result = null
+        var BUFSIZE = 256
+        var buf = Buffer.alloc(BUFSIZE)
+        var bytesRead = fs.readSync(process.stdin.fd, buf, 0, BUFSIZE, -1)
         if (bytesRead === 0) {
-          return null;
+          return null
         }
-        result = buf.slice(0, bytesRead);
-        tty.input = Array.from(result);
+        result = buf.slice(0, bytesRead)
+        tty.input = Array.from(result)
       }
-      return tty.input.shift();
+      return tty.input.shift()
     },
     put_char(tty, val) {
       try {
-        if(val !== null){
-          tty.output.push(val);
+        if (val !== null) {
+          tty.output.push(val)
         }
         if (val === null || val === 10) {
-          process.stdout.write(Buffer.from(tty.output));
-          tty.output = [];
+          process.stdout.write(Buffer.from(tty.output))
+          tty.output = []
         }
-      } catch(e){
-        console.warn(e);
+      } catch (e) {
+        console.warn(e)
       }
     },
     flush(tty) {
       if (!tty.output || tty.output.length === 0) {
-        return;
+        return
       }
-      stream.write(Buffer.from(tty.output));
-      tty.output = [];
-    }
-  };
+      stream.write(Buffer.from(tty.output))
+      tty.output = []
+    },
+  }
 }
 
-function setupStreams(FS, TTY){
-  let mytty = FS.makedev(FS.createDevice.major++, 0);
-  let myttyerr = FS.makedev(FS.createDevice.major++, 0);
+function setupStreams(FS, TTY) {
+  let mytty = FS.makedev(FS.createDevice.major++, 0)
+  let myttyerr = FS.makedev(FS.createDevice.major++, 0)
   TTY.register(mytty, make_tty_ops(process.stdout))
   TTY.register(myttyerr, make_tty_ops(process.stderr))
-  FS.mkdev('/dev/mytty', mytty);
```

Comment:
You use some automated formatter that removes semicolons? I think human-written Javascript should always have semicolons because otherwise the parser can sometimes decide to join multiple lines into one in surprising ways.

Reply:
Prettier.

It's pretty much standard in frontend JavaScript, I think it takes care of semicolons when they're actually necessary, but you can also configure it it force them I think.

Reply:
Huh, I thought with the standard config prettier includes semicolons? But I guess it's a matter of preference then.

Reply:
You were right, by default it does include them, I had a custom setting.

I've reverted this change and put back the commas.